.github/workflows/block-autosquash-commits.yml

```diff
@@ -1,4 +1,4 @@
-on: pull_request
+on: [pull_request, merge_group]
```

Comment:
Shouldn't this also be limited to `checks_requested`?

Reply:
`checks_requested` is the only type available